InAppPurchasesAtlasAppServices/README.md

```diff
@@ -0,0 +1,72 @@
+# Integrating In-App Purchases with Atlas App Services using Flexible Sync
+
+This example contains a skeleton app to be used as a reference on how to In App Purchase with a seamless integration in App Services using the [Swift SDK](https://www.mongodb.com/docs/realm/sdk/swift/#realm-swift-sdk).
+In this project you will be able to see an example of the following use case:
+* Authenticate users using the Swift SDK.
```

Comment:
I'm missing some point explaining with integrate with store kit and what we do with it

Reply:
I've updated it now with more details. Let me know if it's better or if you want more detailed information.

Reply:
It's ok, I thinks that's all the comments solved

Reply:
Nice, thank you for all the help and the review! I'll go ahead and squash and merge